lib/screens/home/widgets/selectedIconTheme.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:flutter/material.dart';
```

Comment:
this is not technically a widget. create a folder in utils, call it theme and place it there.

Reply:
done!